### ISSUE TITLE
Sails server on a different port than client app

### ISSUE THREAD

**Issue by collinforrester** [balderdashy/sails.io.js#30]
Hey,  we use sails primarily for an API so none of our static content is ever hosted on the same domain (since it could come from our web app or our mobile app).

I'm having trouble getting sockets working and to even connect.

Just trying locally first I have the server running on http://localhost:1337.
I have the client running locally on http://localhost:5000 and even if I have the origin set to * or *:* I still get `XMLHttpRequest cannot load http://localhost:1337/socket.io/1/?__sails_io_sdk_version=0.10.0&__sails_io_sdk_platform=browser&__sails_io_sdk_language=javascript&t=1412971419967. No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://localhost:5000' is therefore not allowed access. The response had HTTP status code 403.`

What am I doing wrong?

**Comment by mickyginger** [comment]
I have tried that, and I still get the same issue. Sails v0.10.5, sails.io.js v0.11.4

**Comment by clappingonehand** [comment]
I had the same issue. Sails v0.10.5, sails.io.js v0.11.4. Could it be related to [this issue](https://github.com/Automattic/socket.io-client/issues/641)?

**Comment by sailsbot** [comment]
Thanks for posting, @collinforrester.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
